types/express/express-tests.ts

```diff
@@ -131,12 +131,36 @@ namespace express_tests {
         req.params.length; // $ExpectType number
     });
 
+    // Params can used as an array
```

Comment:
```suggestion
    // Params can used as an array and can be specified via an explicit param type (express-serve-static-core)
```

Reply:
LGTM, I will push a new commit to add this comment.